docs/development.md

```diff
@@ -0,0 +1,74 @@
+## Development
+
+## Application Installation
+
+Open Truss is a full-stack framework installable into existing applications. Open Truss comes with scripts to generate client and server files, adding them to your application.
+
+OT supports the following applications:
+- NextJS
+
+### NextJS
+
+This assumes a standard typescript NextJS app structure with a `src` directory.
+
+1. Run `npm install @open-truss/open-truss`
+1. Copy [open-truss.conf.default.yaml](../packages/open-truss/nextjs/open-truss.conf.default.yaml) into NextJS root and name `open-truss.conf.yaml`
+1. Add `"ot:setup": "open-truss setup"` to package.json and run
+  **Important** - this adds files to your NextJS application, overwriting them if they exist. These are namespaced (e.g. `ot`) and should not conflict with your application-specific files. See [bin/setup](../packages/open-truss/bin/setup) for the full list of files added.
```

Comment:
The wording here is a tad confusing. Maybe mention namespacing before the overwriting warning since overwriting is less likely due to namespacing?

Reply:
```suggestion
  **Important** - this adds open-truss namespaced files to your NextJS application, overwriting them if they exist. These are in prefixed with `open-truss` or in directories such as `ot` and should not conflict with your application-specific files. See [bin/setup](../packages/open-truss/bin/setup) for the full list of files added.
```